docs/build-from-scratch.md

```diff
@@ -34,22 +34,34 @@ Download the archive named `Source code` from the [release page][22] and extract
 Note that the Makefile targets assume building from a git repository. When building from an archive, you will be limited to the `go build` commands described below.
 
 ## Build
+There are a number of different ways to build `velero` depending on your needs. This section outlines the main possibilities.
 
-You can build your Velero image locally on the machine where you run your cluster, or you can push it to a private registry. This section covers both workflows.
-
-Set the `$REGISTRY` environment variable (used in the `Makefile`) to push the Velero images to your own registry. This allows any node in your cluster to pull your locally built image.
+To build the `velero` binary on your local machine, compiled for your OS and architecture, run:
+```bash
+go build ./cmd/velero
+```
+or:
+```bash
+make local
+```
 
-In the Velero root directory, to build your container with the tag `$REGISTRY/velero:$VERSION`, run:
+The latter will place the compiled binary under `$PWD/_output/bin/$GOOS/$GOARCH`, and will splice version and git commit information in so that `velero version` displays proper output.
```

Comment:
Is it worth noting that `make local`'s version info will get used by `velero install` to determine what version of the image gets used?